src/transformers/integrations/executorch.py

```diff
@@ -0,0 +1,133 @@
+from packaging import version
+
+from transformers.testing_utils import (
+    is_torch_available,
+    require_torch,
+)
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        PreTrainedModel,
+        StaticCache,
+    )
+
+
+@require_torch
+class TorchExportatibleModuleWithStaticCache(torch.nn.Module):
```

Comment:
nitpick: is there a typo? exportatible? :)

Reply:
Aha, typo. Fixed now.